config/nrfconnect/chip-module/Kconfig.defaults

```diff
@@ -108,9 +108,19 @@ config BT_PERIPHERAL_PREF_MIN_INT
 config BT_PERIPHERAL_PREF_MAX_INT
     default 36
 
+# Increase BT timeout to 5 s to improve connection reliability and avoid fast drop outs.
```

Comment:
Nit: Maybe it's worth adding an explanation why the BT timeout was increased to 5 s while the default value of the config is set to 500.

Reply:
That's explained in this Kconfig definition, so I didn't want to copy the help text 
"config BT_PERIPHERAL_PREF_TIMEOUT
	int "Peripheral preferred supervision timeout in 10ms units""

Nevertheless I can add it, but it would require updating upstream first, so I can do that by the way of doing some other changes.